syntax = "proto3";
package adet;

option optimize_for = LITE_RUNTIME;

message Details
{
    int32 time = 1;
    int32 value1 = 2;
    int32 value2 = 3;
    string description = 4;
}

message SensorInfo
{
    string name = 1;
    Details data = 2;
}